template.yaml

```diff
@@ -383,7 +457,8 @@ Resources:
           "redirectPathSignOut": "${RedirectPathSignOut}",
           "redirectPathAuthRefresh": "${RedirectPathAuthRefresh}",
           "cookieSettings": ${CookieSettings},
-          "httpHeaders": ${HttpHeaders}
+          "httpHeaders": ${HttpHeaders},
+          "clientSecret": "${ClientSecretRetrieval.ClientSecret}"
```

Comment:
Small favor (sorry for the nit picking) can we move it directly under clientId everywhere?

Reply:
Just to clarify, you mean update all `clientSecret` calls to `clientId` correct? Want to ensure my understanding.

Reply:
Nah I meant put the clientSecret property right below, following, the clientId property please, so they're nicely together like the siblings they are :)

Reply:
Appreciate the clarification, Updated.